pkg/plugins/models.go

```diff
@@ -12,6 +12,7 @@ const (
 )
 
 var (
+	// TODO: Move these?
```

Comment:
Not for this PR but it may be worth to remove or rename these since external packages see at the same level `plugins.ErrPluginNotRegistered` and `plugins.ErrPluginNotInstalled` which can be confusing from a user perspective. WDYT?

Reply:
Done in https://github.com/grafana/grafana/pull/75002